scripts/pi-hole/help.js

```diff
@@ -1,6 +1,6 @@
 $( "#flush" ).click(function() {
     if (confirm("Are you sure you want to flush the Pi-hole log file?")) {
-        document.location.href="help.php?flush";
+        document.location.href="../../help.php";
```

Comment:
This should be fine the way it was previously, since document.location.href doesn't change when the location of the js changes.